documentation/asciidoc/computers/camera/libcamera_vid.adoc

```diff
@@ -106,9 +106,11 @@ Using `libcamera-vid` to capture high framerate video (generally anything over 6
 1. The https://en.wikipedia.org/wiki/Advanced_Video_Coding#Levels[H.264 target level] must be set to 4.2 with the `--level 4.2` argument.
 2. Software colour denoise processing must be turned off with the `--denoise cdn_off` argument.
 3. For rates over 100 fps, disabling the display window with the `-n` option would free up some additional CPU cycles to help avoid frame drops.
-4. It is advisable to set `force_turbo=1` in `/boot/config.txt` to ensure the CPU clock does not get throttled during the video capture. See https://www.raspberrypi.com/documentation/computers/config_txt.html#force_turbo[here] for further details.
+4. It is advisable to set `force_turbo=1` in `/boot/firmware/config.txt` to ensure the CPU clock does not get throttled during the video capture. See https://www.raspberrypi.com/documentation/computers/config_txt.html#force_turbo[here] for further details.
 5. Adjust the ISP output resolution with `--width 1280 --height 720` or something even lower to achieve your framerate target.
-6. On a Pi 4, you can overclock the GPU to improve performance by adding `gpu_freq=550` or higher in `/boot/config.txt`.  See https://www.raspberrypi.com/documentation/computers/config_txt.html#overclocking[here] for further details.
+6. On a Pi 4, you can overclock the GPU to improve performance by adding `gpu_freq=550` or higher in `/boot/firmware/config.txt`.  See https://www.raspberrypi.com/documentation/computers/config_txt.html#overclocking[here] for further details.
```

Comment:
Eek. Not your fault @nathan-contino , but could you convert these https://www.raspberrypi.com/documentation/ absolute-links into the appropriate relative-link xrefs please?

Reply:
Please don't use "here" as a link word. The rule is that text should read properly, and be a real English sentence, without the link. Then insert the link into the sentence. There are a number of instances were "here" is used in the documentation (mostly due to third-party contributions before I started). But we're trying to strip them out.

Reply:
So the xref here would be, `xref:config_txt.adoc#overclocking[link text]` (note the `.adoc` there!) as it's linking to another file inside the `computers/` section, if you were linking out to something in `accessories/` it'd be `xref:../accessories/foo.adoc[link text]`.

Reply:
...and this is why #3096 got opened. 😩

Reply:
> ... into the appropriate relative-link xrefs

I added a note to #3044